rmf_fleet_adapter/schemas/event_description_PerformAction.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/open-rmf/rmf_ros2/main/rmf_fleet_adapter/schemas/event_description_PerformAction.json",
+  "title": "PerformAction event Description",
+  "description": "Description schema for the perform_action category",
+  "type": "object",
+  "properties": {
+    "category": {
+      "description": "A string that describes the nature of this action. It should match that passed into FleetUpdateHandle::add_performable_action()",
+      "type": "string"
+    },
+    "description": {
+      "description": "A dictionary describing the action to be performed.",
+      "type": "object"
```

Comment:
Let's not force this to be a dictionary. Maybe a system integrator would like this to be an array. We can just remove this `"type": "object"` specification and leave it ambiguous.

I'd also recommend changing `description` to something like `"Additional information that will be passed along to the action executor."